.github/workflows/proof.yml

```diff
@@ -8,6 +8,19 @@ on:
   pull_request_target:
     types: [labeled, synchronize]
```

Comment:
Is `opened`, `reopened` not used on purpose?

Reply:
I think we might actually want to remove at least `opened` from all of them, because when it's opened it's not going to have any of the labels on it, and I don't think we want to automatically re-trigger if it's only reopened without any other action. I just forgot about this again after I did the proof test workflow.

Removing `opened` also removes it from the list of checks initially, i.e. currently the hw tests always show up, but as cancelled. If they are not triggered on `open`, I don't think they show up in the list until the label is activated the first time.